packages/fxa-payments-server/src/components/DialogMessage/index.stories.tsx

```diff
@@ -16,7 +16,11 @@ storiesOf('components/DialogMessage', module)
       <DialogToggle>
         {({ dialogShown, hideDialog }) =>
           dialogShown && (
-            <DialogMessage onDismiss={hideDialog}>
+            <DialogMessage
```

Comment:
If you wanna be extra cool you could refactor this to the new storybook format 🙈

Reply:
I'll keep this in mind and see how far I get with my other two tickets. I know I'll eventually have to refactor this. 🙃